### PR TITLE
qemu_guest_agent: add steps for case 'check_fsfreeze_hook_script'

### PR DESCRIPTION
**Pull request by 6-dehan**
add checkpoint and steps for case 'check_fsfreeze_hook_script', since customer case ask to check the logfile of hook_script

ID: 3120
Signed-off-by: demeng [demeng@redhat.com](mailto:demeng@redhat.com)

### CODE REVIEW

File: qemu/tests/qemu_guest_agent.py [hunk 1]
```diff
@@ -3124,6 +3132,33 @@ def log_check(action):
         )
         cmd_get_log_path = "cat %s |grep ^LOGFILE" % hook_path
         log_path = session.cmd_output(cmd_get_log_path).strip().split("=")[-1]
+
+        error_context.context(
+            "Check Check if log_path is the type of virt_qemu_ga_log.", LOG_JOB.info
```
Comment (leidwang) [inline qemu/tests/qemu_guest_agent.py:3137]:
```suggestion
            "Check if log_path is the type of virt_qemu_ga_log.", LOG_JOB.info
```

File: qemu/tests/qemu_guest_agent.py [hunk 2]
```diff
@@ -382,6 +382,12 @@ def setup(self, test, params, env):
                 LOG_JOB.info("qemu-ga service is not running.")
                 self.gagent_start(session, self.vm)
 
+            error_context.context(
+                "Set selinux policy to 'Enforcing' mode in" " guest.", LOG_JOB.info
```
Comment (leidwang) [inline qemu/tests/qemu_guest_agent.py:386]:
```suggestion
                "Set selinux policy to 'Enforcing' mode in guest.", LOG_JOB.info
```

**Review by leidwang — APPROVED** [review]
LGTM

**Comment by 6-dehan** [comment]
@vivianQizhu please help to review and merge this patch. thanks

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**